src/main/java/io/github/hildi/can/exceptions/FileIsNotExistsException.java

```diff
@@ -0,0 +1,10 @@
+package io.github.hildi.can.exceptions;
+
+/**
+ * Created by Serhii Hildi on 24.03.19.
+ */
+public class FileIsNotExistsException extends RuntimeException {
```

Comment:
`file is not exists` is not correct, from the grammar perspective. 
the correct form is `file does not exist`

please change the name of exception and exception message